cooking-app/src/components/Content/AddRecipes/AddRecipe.js

```diff
@@ -0,0 +1,117 @@
+import { addDoc, Timestamp, collection } from "firebase/firestore";
+import { db } from "../../../api/firebase";
+import { RecipeForm } from "./RecipeForm copy 2";
+import { useEffect, useState } from "react";
+import { storage } from "../../../api/firebase";
+import { ref, uploadBytes } from "firebase/storage";
+import { v4 } from "uuid";
+import {
+  urlStorage,
+  urlStorageCD,
+  folderStorage,
+  recipesCollection,
+} from "../../../api/firebaseIndex";
+import { textsRecipe } from "./RecipeHelper";
+import {
+  firestoreErrorsCodes,
+  storageErrorsCodes,
+} from "../../../api/firebaseIndex";
+
+export const AddRecipe = () => {
+  const defaultRecipe = {
+    title: "",
+    time: "",
+    portion: "",
+    ingredients: "",
+    describe: "",
+    url: [],
+    categories: [],
+    recipeTimestamp: Timestamp.fromDate(new Date()).toDate(),
+    author: " {name, email} z Context (jak zmergujemy)",
```

Comment:
" {name, email} z Context (jak zmergujemy)" takie informacje dodawajcie w komentarzach.

Reply:
zrobiony komentarz   // TO DO